calculator.html

```diff
@@ -72,14 +71,13 @@
           </li>
         </ul>
       </form>
-      <a href="#" data-bind="click: removeCalculator, visible: calculators().length > 1">Remove</a>
+      <a href="#" data-bind="click: $parent.removeCalculator, visible: $parent.calculators().length > 1">Remove</a>

```

Comment:
These things are not global anymore. So we have to reference them from the parent of the binding context (`new App()`)